symfony/webpack-encore-bundle/1.0/config/packages/webpack_encore.yaml

```diff
@@ -2,7 +2,12 @@ webpack_encore:
     # The path where Encore is building the assets.
     # This should match Encore.setOutputPath() in webpack.config.js.
     output_path: '%kernel.project_dir%/public/build'
+    # If multiple builds are defined (as shown below), you can disable the default build:
+    # output_path: false	    # output_path: false
```

Comment:
why is it duplicated ?

Reply:
lack off coffee, fat fingers, and old mac laptop sticky keys...